### PR TITLE
desktop view

### PR DESCRIPTION
**Pull request by maths29**
In this project I:
- Added a desktop version to the project
- Added modifications to style.css

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,492 +1,941 @@
...
+    background-color: #fff;
+    padding-top: 17px;
+    padding-right: 100px;
+  }
+
+  .head-text {
+    width: 100%;
+    margin: 40px 0;
+    margin-top: -20px;
+  }
```
Comment (ABDULALI3468) [inline style.css]: Please consider making the design exactly inaccordanc with the Figma template currently the positions of elements in both of the designs are very different. 

> Remember don't you can target different desktop screens like in ranges as currently if I see your design at 800px it is broken at many places as you added long margins to individual elements so consider focusing on that also.

### VIDEO DEMONSTRATION:-

https://user-images.githubusercontent.com/94223367/236555543-0813f84b-5b68-4663-ad89-70d9837c5c10.mp4

### FIGMA DESIGN:-

image

File: style.css [hunk 2]
```diff
@@ -1,492 +1,941 @@
...
+    margin-top: 15px;
+    margin-bottom: 1px;
+    margin-left: -17px;
+  }
+
+  .cap {
+    letter-spacing: 2px;
+    margin-left: 14px;
+    margin-top: 30px;
+  }
```
Comment (ABDULALI3468) [inline style.css]: Please consider using the same letter spacing as used in the Figma design to match the design and to meet the project requirements.

### REMEMBER:-

THE GOAL OF THE PROJECT TO MAKE A LOOK ALIKE DESIGN OF THE TEMPLATE!

File: style.css [hunk 3]
```diff
@@ -1,492 +1,941 @@
...
+    border: 1px solid #dfe1e6;
+    padding: 14px;
+    margin-top: 15%;
+    background: #fff;
+    height: auto;
+    border-radius: 0 70px 0 0;
+    padding-bottom: 120px;
+    width: 100%;
+    gap: 0%;
+  }
```
Comment (ABDULALI3468) [inline style.css:736]: Please consider making the design exactly inaccordanc with the Figma template currently the positions of elements in both of the designs are very different. 

> Remember don't you can target different desktop screens like in ranges as currently if I see your design at 800px it is broken at many places as you added long margins to individual elements so consider focusing on that also.

### VIDEO DEMONSTRATION:-



https://user-images.githubusercontent.com/94223367/236557164-283ba3f6-d5c3-47a6-b8d8-2354716e6476.mp4


### FIGMA DESIGN:-

image

File: style.css [hunk 4]
```diff
@@ -1,492 +1,941 @@
...
+  .contact-space {
+    margin-top: 30px;
+    min-height: 80vh;
+    background: url(./Img/contact\ form\ background\ shapes.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: contain;
+    text-align: center;
+    width: 100%;
+  }
```
Comment (ABDULALI3468) [inline style.css:888]: The background images are not showing in the design please consider making the design in accordance with the template to meet the project requirements ✅ 

image

File: style.css [hunk 5]
```diff
@@ -1,492 +1,941 @@
...
 
-.inner {
-  padding: 40px;
+.firstS1 .squire {
+  font-size: 32px;
+  font-weight: 700;
+  color: #172b4d;
+  margin-top: 9px;
+  margin-left: -200px;
 }
```
Comment (ABDULALI3468) [inline style.css]: Remember the mobile version still needs to look perfect as it was before currently there is a different in position with respect to design please rectify it to meet the design guidelines.

image

File: index.html [hunk 6]
```diff
@@ -12,11 +12,21 @@
...
+        <div class="nav-top">
+          <div class="shape">
+            <a href="#">portfolio</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+          </div>
+          <img src="img/Union.png" />
+        </div>
+        
       </nav>
```
Comment (ABDULALI3468) [inline index.html:24]: Please consider positioning the logo of the navbar similar to the design to meet the project requirements.

### YOUR DESIGN:-

image

### FIGMA DESIGN:-

image

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @maths29 , 



Indeed you did an exceptional but still, there is room for changes to complete this project. 💯


## Highlights
💹 Well-implemented design. 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

Please consider always using the name of your feature branch as a way to highlight what you did in it (e.g. 'created_sidebar' or 'added_comments_section')  In this case, your branch name is `feature-b` which is not professional.

>You can name it `feature-1/portfolio-desktop-version` or something else that describes what you did in the branch.

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -307,116 +316,112 @@ <h2 class="squire" id="caption">
...
+      </div>
+      <section class="aboutSection">
+        <div class="sec">
+          <div class="sec1">
+            <h3 class="sect">About Myself</h3>
+            <p class="abt-sec">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hesitate to contact me.
```
Comment (OyePriscilla) [inline index.html:328]: - [ ] As suggested earlier, kindly work more on the about me section in accordance to the given template. 👇 
## Your Design
Untitled

File: style.css (hunk 5 above)
Comment (OyePriscilla) [reply style.css]: - [ ] Please kindly ensure that the mobile version still look as expected as raised earlier. 🙏 
## Your Design
Untitled

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @maths29 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
App responsiveness ✔️ 
Uses both Flexbox and Grid in order to layout the elements on the page ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @maths29 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
App responsiveness ✔️ 
Uses both Flexbox and Grid in order to layout the elements on the page ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @maths29, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 


## Optional suggestions

There is still room for improvement but you put enough effort into this project. It is approved and if you want to make it better here is what you can do:    

- [OPTIONAL] Please consider aligning the About section title with the paragraph, I see that you use a lot of fixed margins across your entire stylesheet which makes getting this done a hard task, my suggestion is that you avoid using them unless they are necessary. 🤓 🔧 

Screenshot from 2023-05-06 12-39-28




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**